bin/database-ready.rb

```diff
@@ -31,8 +33,22 @@ def run!
   def can_connect?
     loop do
       puts "Attempting to connect with #{model}"
-      num = model.connection.execute('select 2*3').values.flatten.first.to_i
-      return unless num != 6
+
+      begin
+        puts 'Executing test query...'
+        result = Timeout.timeout(30) do
+          model.connection.execute('select 2*3')
+        end
+        puts 'Query executed successfully, processing result...'
+
+        num = result.values.flatten.first.to_i
+        puts "Query result: #{num}"
+        return unless num != 6
```

Comment:
Any issue removing the double negative here? It feels like it's adding extra complexity to reading what the code is doing.
```suggestion
        return if num == 6
```